src/utils/src/FileLogger.c

```diff
@@ -165,7 +165,7 @@ STATUS createFileLogger(UINT64 maxStringBufferLen, UINT64 maxLogFileCount, PCHAR
     UINT64 charWritten = 0, indexFileSize = KVS_COMMON_FILE_INDEX_BUFFER_SIZE;
 
     // allocate the struct and string buffer together
-    gFileLogger = (PFileLogger) MEMALLOC(SIZEOF(FileLogger) + maxStringBufferLen * SIZEOF(CHAR));
+    CHK(NULL != (gFileLogger = (PFileLogger) MEMALLOC(SIZEOF(FileLogger) + maxStringBufferLen * SIZEOF(CHAR))), retStatus);
```

Comment:
Just wanted to note that here, the application of MEMALLOC followed by MEMSET to zero out the buffer is potentially a better choice due to the buffer size being allocated being large. This could cause other applications impact as the system zero page pool could get impacted. 

Thanks for the added check